pywal/__main__.py

```diff
@@ -332,6 +343,10 @@ def main():
     util.create_dir(os.path.join(CONF_DIR, "colorschemes/dark/"))
 
     util.setup_logging()
+
+    if show_colorama_warning:
```

Comment:
... otherwise this may fail with variable not set errors.

Reply:
thanks to https://github.com/eylles/pywal16/pull/152/commits/67af8f0662e1528f943dd4e2bd295a41b03c5302 the var is set to false by default